### PR TITLE
Add download task for instrumentation test only

### PR DESCRIPTION
**Pull request by zbynek**
Currently many execute tasks are available for each variant, but only one download task which depends on all the tests, including Robo. I'd like to be able to run download only for the instrumentation tests. This feature request was already mentioned in #16 by @johnfrey99.

The commit adds download task for the top level instrumentation tasks and moves download task creation to `addExecuteAndDownload` so that it's easier to add download tasks for the per-device tasks in the future if needed.

### CODE REVIEW

**Comment by piotrmadry** [comment]
Great! Looks like nice improvement. Can you please resolve conflicts and write tests for those download tasks? @zbynek

File: plugin/src/main/java/com/appunite/firebasetestlabplugin/FirebaseTestLabPlugin.kt [hunk 1]
```diff
@@ -367,11 +356,30 @@ internal class FirebaseTestLabPlugin : Plugin<Project> {
             }
         }
     }
+
+    private fun addExecuteAndDownload(name: String, downloader: CloudTestResultDownloader?, cleanTask: String, taskClosure: Closure<Any?>): Task {
```
Comment (piotrmadry) [inline plugin/src/main/java/com/appunite/firebasetestlabplugin/FirebaseTestLabPlugin.kt]: I think we should stick to `taskClosure: Closure<Task>` I'm not big fan of giving to much space to make a mistake. We can always expand it if needed

**Comment by zbynek** [comment]
@piotrmadry thanks for the feedback, I refactored it a bit to use `taskClosure: Closure<Task>` again without repeating too much code, hope it's fine.

**Comment by zbynek** [comment]
@piotrmadry sorry for nagging, will you have time to review this PR?

**Comment by piotrmadry** [comment]
I'm so sorry @zbynek I'm facing lack of time currently. Looks good for me. I'm going to merge it and release new version. It will be 2.1.0

**Comment by piotrmadry** [comment]
@zbynek https://plugins.gradle.org/plugin/firebase.test.lab

**Pull request merged**